Source/LINQPadDataConnection.cs

```diff
@@ -24,6 +26,7 @@ public LINQPadDataConnection(IConnectionInfo cxInfo)
 				(string?)cxInfo.DriverData.Element(CX.ProviderPath),
 				cxInfo.DatabaseInfo.CustomCxString)
 		{
+			CommandTimeout = cxInfo.DriverData.ElementValueOrDefault(CX.CommandTimeout, str => str.ToInt32() ?? 0, 0);

```

Comment:
Should default be infinite or the usual default of 30s? I'm good with either though. Approving either way.

Reply:
default is 0, which is infinite. we were setting it everywhere except this one place